packages/core/echo/echo-db/src/packlets/database/database-proxy.ts

```diff
@@ -20,7 +20,12 @@ import { ItemManager } from './item-manager';
 
 const FLUSH_TIMEOUT = 5_000;
 
-const BATCH_INTERVAL = 200;
+export const BATCH_COMMIT_AFTER = 200;
+/**
+ * Maximum number of mutations in a batch.
+ * Note: It is used only in auto created batches, if user creates/commits batch outside it will be ignored.
+ */
+const BATCH_SIZE_LIMIT = 64;
```

Comment:
always move consts into an FooOptions type/object that is passed into the constructor -- this allows us to push the options up the tree. And to configure tests, etc.

Reply:
Makes sense, thanks